recipes/raxml-ng/meta.yaml

```diff
@@ -1,22 +1,22 @@
 {% set name = "RAxML-NG" %}
```

Comment:
This is unrelated to the update here, but we could change it along the way. Usually, this name is used in the `url` below, so:
```
{% set name = "raxml-ng" %}
```
And then:
```
source:
  url: https://github.com/amkozlov/raxml-ng/releases/download/{{ version }}/{{ name }}_v{{ version }}_source.zip
```

But the recipe is fine without this change, so feel free to include this suggestion or merge it as is with the `please merge` BiocondaBot comment suggested above.